.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you must not commit files not related to the project. All you have in `.idea` folder are your local Intellij configs and other stuff. Other people have their own and probably different of yours. Such files/folders should be ignored from commit (see https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files)